x-pack/filebeat/module/aws/vpc/config/file.yml

```diff
@@ -0,0 +1,6 @@
+type: log
+paths:
+  {{ range $i, $path := .paths }}
+  - {{$path}}
+    {{ end }}
+exclude_files: [".gz$"]
```

Comment:
The `processors` I was describing would be defined here because they are part of the Beat's configuration.

Reply:
Thanks for pointing it out. I will need to combine both file.yml and s3.yml here to share the same code for processors.